hmcts/div-petitioner-frontend
# [PREVIEW] Div 2815 check payment status

# Description

See Ticket Div-2815

In some case the petitioner could have done a successful payment but CCD has not been updated with it. So when a petitioner login, if the petition is in state Awaiting Payment, the application will check with payment application if a success full payment has been made and update CCD.

Use this PR for testing https://github.com/hmcts/div-petitioner-frontend/pull/321 - this PR has check from ccd activated correctly

Fixes # (issue)

## Type of change


# How Has This Been Tested?

- Submit a petition, make a payment and logout immediately.
- Manually check the payment is successful but CCD is still in Awaiting Payment
- Log in petition
- Check in CCD the State is not Awaiting Payment anymore.


**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
